packages/manager/modules/vps/src/dashboard/migrate/migrate.html

```diff
@@ -0,0 +1,113 @@
+<oui-tile-definition
+    data-term="{{:: 'vps_dashboard_migrate_title'| translate}}"
+>
+    <oui-tile-description>
+        <oui-select-picker
+            class="w-25 mt-3"
+            label="{{:: 'vps_dashboard_migrate_current_range_title' | translate}}"
+            disabled
+        >
+            <oui-select-picker-section>
+                <ul class="list-unstyled">
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_core"
+                        ></span>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_vcore"
+                            data-translate-values="{ count: $ctrl.stateVps.vcore }"
+                        ></span>
+                    </li>
+                    <li>
+                        <span
+                            data-translate="vps_dashboard_migrate_configuration_label_ram"
+                        ></span>
+                        <span
+                            data-ng-bind=":: ($ctrl.stateVps.model.memory / 1024) | cucBytes:0:false:'GB'"
```

Comment:
Maybe we can avoid doing some math in the template and extract the `$ctrl.stateVps.model.memory / 1024`